polygon/websocket/models/common.py

```diff
@@ -9,6 +9,16 @@ class Feed(Enum):
     PolyFeedPlus = "polyfeedplus.polygon.io"
     StarterFeed = "starterfeed.polygon.io"
     Launchpad = "launchpad.polygon.io"
+    BusinessFeed = "business.polygon.io"
+    EdgxBusinessFeed = "edgx-business.polygon.io"
+    DelayedBusinessFeed = "delayed-business.polygon.io"
+    DelayedEdgxBusinessFeed = "delayed-edgx-business.polygon.io"
+    DelayedNasdaqLastSaleBusinessFeed = "delayed-nasdaq-last-sale-business.polygon.io"
+    DelayedNasdaqBasicFeed = "delayed-nasdaq-basic-business.polygon.io"
+    DelayedFullMarketBusinessFeed = "delayed-fullmarket-business.polygon.io"
+    FullMarketBusinessFeed = "fullmarket-business.polygon.io"
+    NasdaqfeedLastSaleBusinessFeed = "nasdaq-last-sale-business.polygon.io"
```

Comment:
we can probably omit _feed_ from these variable names to stay consistent with the delayed names

Reply:
this was just copy/paste from the go client, good point 👍

Reply:
oh wait, my comment here was unclear. I think we should still append "Feed" to every one of these. 

But i think we should replace "Nasdaqfeed" with just "Nasdaq" for the nasdaq feeds

Reply:
why? none of the original feed names include `Feed` other than `Polyfeed(Plus)` and `StarterFeed` which is just part of the product name in those cases

Reply:
Ah okay, I thought they all had Feed at the end, nevermind about that then. 

But I still think it makes sense to replace "Nasdaqfeed" with just "Nasdaq" for the nasdaq feeds

Reply:
already done, see alice's comment